place_tracker/lib/place_list.dart

```diff
@@ -22,8 +22,7 @@ class PlaceListState extends State<PlaceList> {
   void _onPlaceChanged(Place value) {
     // Replace the place with the modified version.
     final List<Place> newPlaces = List.from(AppState.of(context).places);
-    final int index =
-        newPlaces.indexWhere((Place place) => place.id == value.id);
+    final int index = newPlaces.indexWhere((place) => place.id == value.id);
```

Comment:
nit: Could lose the type annotations here and on 24?

Reply:
done